package.json

```diff
@@ -104,5 +104,8 @@
   "repository": {
     "type": "git",
     "url": "git+https://github.com/nrkno/core-icons.git"
+  },
+  "dependencies": {
+    "svg2vectordrawable": "^2.9.1"
```

Comment:
Please lock this to an explicit version using `npm i -E svg2vectordrawable` (lockfile should update too) to avoid possible different versions being used to build, between two unrelated commits to `master`.